### PR TITLE
refactor(GratefulPage/header) Make header its own

### PR DESCRIPTION
**Pull request by dope**

### CODE REVIEW

File: src/styles/main.css [hunk 1]
```diff
@@ -184,4 +193,14 @@ html {
...
...4.3c-9.2-14.5-22-23-33.4-28.1-6.6-3-14.2.2-17.4 6.7a482.8 482.8 0 0 0-35.5 96 4.5 4.5 0 0 0 7.1 4.7c27-20.1 56.1-47 77.6-68.2 3-3 3.8-7.5 1.6-11zM114 180.6c-24-27-69.8-67.6-93.3-87.2-6.2-5.2-14.5-5.2-19.6 1-5.7 7-12.4 16.8-17.3 26-6.4 11.7.4 24 12.7 29.3 34.7 15.2 73.3 30.8 106.5 44 12.7 5 20-2.9 11-13zM132 144a528.3 528.3 0 0 0-27.5-104 18.4 18.4 0 0 0-27.3-9.3 152.7 152.7 0 0 0-29.6 23.5c-7.4 7.7-6.8 19.3-.2 27.8 14.6 18.8 35 40 73.6 68.6a7 7 0 0 0 11-6.6zM-29.8 217.6c20.9 1.9 44 2.6 62.5 2 14.4-.4 17.1-13 4.4-19.8a1043 1043 0 0 0-57.3-28.2c-8.9-4-19-.1-21.8 9.2a68 68 0 0 0-3 22.2c.4 8 7.2 13.8 15.2 14.6zM153.6 15.5c7.2 16.4 9.5 60.3 10.2 86.2.1 5-6.2 6.6-8.2 2-12.7-28-32.8-80.3-12.9-91.8 4-2.4 9-.7 10.9 3.6zM357.1 88.1a382.2 382.2 0 0 0-60.4 56.2c-3 3.5.1 8.5 4.6 7.2a685.6 685.6 0 0 0 72.3-25c6.4-2.8 9.6-10 6.9-16.5-3.1-7.4-6.8-14-10.1-19.3a9.2 9.2 0 0 0-13.3-2.6zM278.1 158.7a193.4 193.4 0 0 0-34.3 30.6c-3.5 3.8.4 8.6 4.8 5.8 12.9-8.1 28.7-19.4 36.4-30.2 3.8-5.3-1.4-9.7-6.9-6.2zM98.4 214.2c-18.9-6-35-10.7-37-3-2 8 15.7 14.4 36.9 11 4.6-.7 4.6-6.6.1-8zM177.3 187.6c-.3-14.6-3.7-30.6-6.8-42.6-1.3-4.9-7.3-4.3-7.7.7-1 14.3-1.5 33.4 2 46.3 2.7 10.2 12.8 6.2 12.5-4.4z'/%3E%3C/svg%3E");
```
Comment (dope) [inline src/styles/main.css:199]: @glemmaPaul Will potentially add a `images` section to stores these SVGs

File: src/pages/GratefulPage.js [hunk 2]
```diff
@@ -1,47 +1,60 @@
...
+
+const emojis = [
+  `😌`,
+  `💃`,
+  `🙏`,
+  `🎉`,
+]
+
+// Generate random emoji
+const emoji = emojis[~~(emojis.length * Math.random())];
```
Comment (glemmaPaul) [inline src/pages/GratefulPage.js:23]: omg what did you do, it looks amazing but never saw `~~`

Comment (glemmaPaul) [reply src/pages/GratefulPage.js:23]: That floors it to number?

File: src/pages/GratefulPage.js [hunk 3]
```diff
@@ -1,47 +1,60 @@
...
 function GratefulPage() {
-  const [entries, setEntries] = useState(null)
-  const [user, setUser] = useState(null)
+  const [entries, setEntries] = useState(null);
+  const [user, setUser] = useState(null);
 
   useEffect(() => {
-    firebase.auth().onAuthStateChanged(setUser)
-  })
+    firebase.auth().onAuthStateChanged(setUser);
```
Comment (glemmaPaul) [inline src/pages/GratefulPage.js:38]: excuse me

File: src/pages/GratefulPage.js [hunk 4]
```diff
@@ -1,47 +1,60 @@
...
-  })
+  });
 }
 
-
 function GratefulPage() {
-  const [entries, setEntries] = useState(null)
-  const [user, setUser] = useState(null)
+  const [entries, setEntries] = useState(null);
+  const [user, setUser] = useState(null);
```
Comment (glemmaPaul) [inline src/pages/GratefulPage.js:35]: excuse me

File: src/pages/GratefulPage.js [hunk 5]
```diff
@@ -1,47 +1,60 @@
...
     created: new Date()
-  })
+  });
 }
 
-
 function GratefulPage() {
-  const [entries, setEntries] = useState(null)
-  const [user, setUser] = useState(null)
+  const [entries, setEntries] = useState(null);
```
Comment (glemmaPaul) [inline src/pages/GratefulPage.js:34]: excuse me

File: src/pages/GratefulPage.js [hunk 6]
```diff
@@ -1,47 +1,60 @@
...
-    firebase.auth().onAuthStateChanged(setUser)
-  })
+    firebase.auth().onAuthStateChanged(setUser);
+  });
 
   // When user and entries are both truthy, save the items in firebase with the user id
   useEffect(() => {
     if (user && entries) {
-      saveGratitudeLog(user.uid, entries).then(() => console.log("saved"))
+      saveGratitudeLog(user.uid, entries).then(() => console.log("saved"));
```
Comment (glemmaPaul) [inline src/pages/GratefulPage.js:44]: excuse?

File: src/pages/GratefulPage.js [hunk 7]
```diff
@@ -1,47 +1,60 @@
...
     }
-  }, [user, entries])
-
+  }, [user, entries]);
```
Comment (glemmaPaul) [inline src/pages/GratefulPage.js:46]: excuse?

**Review by glemmaPaul — APPROVED** [review]
Looks good!

File: src/pages/GratefulPage.js (hunk 2 above)
Comment (dope) [reply src/pages/GratefulPage.js:23]: Yeah it converts the operand to the closest integer

**Pull request merged**